docker/Dockerfile

```diff
@@ -54,7 +54,7 @@ FROM ${RUNTIME_IMAGE} AS runtime
 # Ref: https://github.com/hexops/dockerfile/blob/main/README.md#do-not-use-a-uid-below-10000
 ARG UID=10001
 ARG USER_NAME=celestia
-ENV CELESTIA_HOME=/home/${USER_NAME}
+ENV CELESTIA_APP_HOME=/home/${USER_NAME}/.celestia-appd
```

Comment:
This doesn't align with https://github.com/celestiaorg/celestia-app/blob/main/app/init.go#L18 which is a little confusing.

```diff
- celestia-appd
+ celestia-app
```

Reply:
Ohhh good catch.

Reply:
Can you fix @damiannolan? All instances of `.celestia-appd` actually